app/src/main/java/org/oppia/android/app/home/HomeActivity.kt

```diff
@@ -49,7 +49,11 @@ class HomeActivity :
       supportFragmentManager.beginTransaction().remove(previousFragment).commitNow()
     }
     val dialogFragment = ExitProfileDialogFragment
-      .newInstance(isFromNavigationDrawer = false)
+      .newInstance(
+        isFromNavigationDrawer = false,
+        isAdministratorControlsSelected = false,
```

Comment:
Rather than storing the "from" might it be cleaner for us to represent the 'behavior' that we desire from ExitProfileDialogFragment? Specifically, I think it's a dependency inversion for something like ExitProfileDialogFragment to specialize behavior based on certain origins. It may be the case that certain origins should drive certain reset behavior (e.g. "set selected tab to X" or "reset to current selection" which is a generic concept rather than tying to specific flows). Does such a general case exist here?

Reply:
This is really a good point, rather than storing `isFromNavigationDrawer` we could store something like `restoreLastCheckedItem` is true or not.
@BenHenning Can you please confirm if you meant the same?

Reply:
Yes, that sounds much cleaner @prayutsu. Good name.

Reply:
Done.